src/helpers/generateModel.ts

```diff
@@ -33,16 +33,29 @@ export const generateModel = (model: DMMF.Model, config: Config) => {
     const dbName = typeof field.dbName === "string" ? field.dbName : null;
 
     if (field.kind === "enum") {
+      const type = field.isList
+        ? ts.factory.createTypeReferenceNode(
+            ts.factory.createIdentifier("EnumArray"),
+            [
+              ts.factory.createTypeReferenceNode(
+                ts.factory.createIdentifier(field.type),
+                undefined
+              ),
+            ]
+          )
+        : ts.factory.createTypeReferenceNode(
```

Comment:
this `ts.factory.createTypeReferenceNode(` can also be reused above to improve readability.